shanoir-uploader/src/main/java/org/shanoir/uploader/action/ImportFromCsvRunner.java

```diff
@@ -453,25 +467,64 @@ protected boolean searchField(String searchedElement, String filter) {
 		if (StringUtils.isBlank(searchedElement)) {
 			return false;
 		}
-		if (StringUtils.isBlank(filter) || filter.equals(WILDCARD)) {
+		if (StringUtils.isBlank(filter) || filter.equals("*")) {
 			return true;
 		}
-		String[] filters = filter.split(";");
+		String[] filters;
 		boolean valid = true;
-		for (String filterToApply : filters) {
-			// NB: we choose to use AND instead of OF between filters.
-			// This way we are more restrictive
-			// If you want more acquisitions, you have to duplicate the line in the CSV with other filters
-
-			if (filterToApply.startsWith("!")) {
-				valid =  valid && !filterWildCard(searchedElement, filterToApply.replaceAll("!", ""));
+		// NB: It is possible to have AND ";" filters OR OR ";;" filters but not both at the same time for the moment.
+
+		if (filter.contains(";;")) {
+			valid = false;
+			filters = filter.split(";;");
+			for (String filterToApply : filters) {
+				if (filterToApply.startsWith("!")) {
+					valid =  valid || !filterWildCard(searchedElement, filterToApply.replaceAll("!", ""));
+				} else {
+					System.err.println(filterToApply + filterWildCard(searchedElement, filterToApply.replaceAll("!", "")));
```

Comment:
Hi Jean-Come, would it be okay to use the logger here, that it comes into the su.log please?